assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -341,6 +340,7 @@
   python3-scons
   java-21-openjdk
   java-21-openjdk-devel
+  oracle-epel-release-el10
```

Comment:
This package can only be installed in Oracle Linux, but current list is for all RHEL 10-based distributions.